tests/check_framework/test_model_checks.py

```diff
@@ -50,6 +50,31 @@ class Meta:
             )
         ])
 
+    @modify_settings(INSTALLED_APPS={'append': 'basic'})
```

Comment:
Test from my comment was just an example, we should test real use case with abstract models, similar to a `test_collision_abstract_model`.